networking/using-ptp.adoc

```diff
@@ -87,6 +87,8 @@ include::modules/cnf-configuring-the-ptp-fast-event-publisher.adoc[leveloffset=+
 
 * For a complete example CR that configures `linuxptp` services as an ordinary clock with PTP fast events, see xref:../networking/using-ptp.adoc#configuring-linuxptp-services-as-ordinary-clock_using-ptp[Configuring linuxptp services as ordinary clock].
 
+include::modules/nw-ptp-deploying-example-consumer-app.adoc[leveloffset=+2]
```

Comment:
Remove this example consumer app cc @aneeshkp will provide some details about subscribing to events as a consumer and calling the events API once subscribed.